src/ActiveField.php

```diff
@@ -342,6 +344,7 @@ public function checkboxList($items, $options = [])
                     Html::addCssClass($wrapperOptions, 'custom-control-inline');
                 }
 
+                $this->addErrorClassIfNeeded($options);
```

Comment:
Should it be here or later on (before line 359)?

Reply:
It should be here, checkboxList contains several values and should be added here for correct display.

Reply:
347 line